ui/components/app/assets/asset-list/asset-list.tsx

```diff
@@ -190,41 +129,13 @@ const AssetList = ({ onClickAsset, showTokensLinks }: AssetListProps) => {
         ) : null
         ///: END:ONLY_INCLUDE_IF
       }
-      <TokenListItem
-        onClick={() => onClickAsset(nativeCurrency)}
-        title={nativeCurrency}
-        // The primary and secondary currencies are subject to change based on the user's settings
-        // TODO: rename this primary/secondary concept here to be more intuitive, regardless of setting
-        primary={
-          showSecondaryCurrency(
-            isOriginalNativeSymbol,
-            useNativeCurrencyAsPrimaryCurrency,
-          )
-            ? secondaryCurrencyDisplay
-            : undefined
-        }
-        tokenSymbol={
-          useNativeCurrencyAsPrimaryCurrency
-            ? primaryCurrencyProperties.suffix
-            : secondaryCurrencyProperties.suffix
-        }
-        secondary={
-          showFiat &&
-          showPrimaryCurrency(
-            isOriginalNativeSymbol,
-            useNativeCurrencyAsPrimaryCurrency,
-          )
-            ? primaryCurrencyDisplay
-            : undefined
-        }
-        tokenImage={balanceIsLoading ? null : primaryTokenImage}
-        isOriginalTokenSymbol={isOriginalNativeSymbol}
-        isNativeCurrency
-        isStakeable={isStakeable}
-        showPercentage
+      <AssetListControlBar
+        setTokenList={setTokenList}
+        showTokensLinks={showTokensLinks}
       />
       <TokenList
-        tokens={tokensWithBalances}
+        nativeToken={<NativeToken onClickAsset={onClickAsset} />}
+        tokens={tokenList}
```

Comment:
Is the native asset meant to always be at the top?  I'm seeing that I have $86 of PEPE and $26 dollars of ETH, but the ETH is at the top when sorting by declining balance


https://github.com/user-attachments/assets/97669428-4b00-4377-aeda-dcca4d5b29a4

Reply:
No this should not be the behavior. Let me double check why this is happening.

Reply:
This should now be resolved here: https://github.com/MetaMask/metamask-extension/pull/27184/commits/a6d153a1c328cd490bc2d78a807145ef725f6926

I felt that this was the easiest way to fix this, without having to go down a refactor of `useCurrencyDisplay`. If there is a cleaner way to do this that I'm missing plz lmk!